submissions/Nemooochka/js-dom/js/script.js

```diff
@@ -0,0 +1,155 @@
+        'director': 'Barry Jenkins',
+        'writers': 'Barry Jenkins (screenplay by), Tarell Alvin McCraney (story by)',
+        'stars': 'Mahershala Ali, Naomie Harris, Trevante Rhodes '
+    },
+    {
+        'year': 'y_2015',
+        'name': 'Spotlight',
+        'poster': 'https://m.media-amazon.com/images/M/MV5BMjIyOTM5OTIzNV5BMl5BanBnXkFtZTgwMDkzODE2NjE@._V1_UX182_CR0,0,182,268_AL_.jpg',
+        'description': 'The true story of how the Boston Globe uncovered the massive scandal of child molestation and cover-up within the local Catholic Archdiocese, shaking the entire Catholic Church to its core.',
+        'director': 'Tom McCarthy',
+        'writers': 'Josh Singer, Tom McCarthy',
+        'stars': 'Mark Ruffalo, Michael Keaton, Rachel McAdams'
+    },
+    {
+        'year': 'y_2014',
+        'name': 'Birdman',
+        'poster': 'https://m.media-amazon.com/images/M/MV5BODAzNDMxMzAxOV5BMl5BanBnXkFtZTgwMDMxMjA4MjE@._V1_UX182_CR0,0,182,268_AL_.jpg',
+        'description': 'A washed-up superhero actor attempts to revive his fading career by writing, directing, and starring in a Broadway production.',
+        'director': 'Alejandro G. Iñárritu',
+        'writers': 'Alejandro G. Iñárritu, Nicolás Giacobone',
+        'stars': 'Michael Keaton, Zach Galifianakis, Edward Norton'
+    }
+];
+
+const initialMovie = data[0];
+const navLink = document.querySelectorAll('.sidebar a');
+const sidebar = document.querySelector(".sidebar");
+const menuChevron = document.querySelector(".menu-chevron");
+
+const toggleSidebar = () => {
+    sidebar.classList.toggle('open');
+};
+
+/* get a year (last part of url) from href */
+const getLastItem = thePath => thePath.substring(thePath.lastIndexOf('/') + 1);
+
+function generateView(obj) {
+    return `<div class="card-poster">
+                <img src="${obj.poster}" alt="${obj.name}">
+            </div>
+            <div class="card-info">
+                <h2>${obj.name}</h2>
+                <div class="details">
+                    <p class="details_description">${obj.description}</p>
+                    <div class="details_director">
+                        <b>Director:</b>
+                        <span class="director">${obj.director}</span>
+                    </div>
+                    <div class="details_writer">
+                        <b>Writers:</b>
+                        <span>${obj.writers}</span>
+                    </div>
+                    <div class="details_stars">
+                        <b>Stars:</b>
+                        <span>${obj.stars}</span>
+                    </div>
+                </div>
+            </div>`;
+}
+
+function renderCard(nodeList) {
+    const cardMovie = document.querySelector('.card');
+    cardMovie.innerHTML = generateView(nodeList);
+}
+
+function updateActiveNav(targetLink) {
```

Comment:
Instead of using this complex function I'd recommend using classNames on your `li` and `a` element. And just use this expression for finding active element.
``` javascript
const activeElement = document.querySelector('.someElement.active');
if (activeElement) {
   activeElement.classList.remove('active');
}
target.closest('.someLiElementClassName').classList.add('active');
```
Only 5 lines of code